docs/deployment/thin-egress-app.md

```diff
@@ -105,3 +115,17 @@ Pass `api_distribution_url` to your `archive` module's `distribution_url` var.
 
 You will also need to configure the `tea_urs_redirect_uri` value as a Redirect
 URI in your app's URS configuration.
+
+### Cumulus Configuration
+
+The default Cumulus module generates a file at `s3://${system_bucket}/distribution_bucket_map.json`. If you've manually deployed TEA outside the `cumulus` module, you'll need to provide this configuration file as part of your Terraform deployment.
+
+The configuration file is a simple json mapping of the form:
+
+```json
+{
+  "bucket_name": "bucket_path_in_TEA"
```

Comment:
I think we need more of an example here like:

`"protected": "MOD09GQ/protectedData/"` whatever makes sense here

Reply:
👍  I'll add a more concrete example

Reply:
fb4864cd5